### PR TITLE
Multicrew and the 'Message received' event

### PR DESCRIPTION
**Pull request by Tkael**
Potential fix for #291.

### CODE REVIEW

**Comment by Tkael** [comment]
@richardbuckle Here's my proposed fix. Please review?

File: JournalMonitor/JournalMonitor.cs [hunk 1]
```diff
@@ -1245,11 +1245,14 @@ public static List<Event> ParseJournalEntry(string line)
...
                                     channel == "friend" ||
                                     channel == "voicechat" ||
-                                    channel == "local"
+                                    channel == "local" ||
+                                    channel == null
                                 )
                                 {
                                     // Give priority to player messages
                                     source = channel == "wing" ? "Wing mate" : "Commander";
+                                    source = channel == null ? "Crew mate" : "Commander";
```
Comment (richardbuckle) [inline JournalMonitor/JournalMonitor.cs]: I think will overwrite the "wing" channel, won't it?
Let's add a unit test for each case.

**Comment by Tkael** [comment]
I read these as 'if channel is wing, source is "Wing mate", else source is "Commander"' and 'if channel is null, source is "Crew mate", else source is "Commander"'. It shouldn't overwrite, but unit tests to check are certainly fine.

**Comment by richardbuckle** [comment]
This, my padawan, is where you learn that how you read the code is neither here nor there, it's how the computer reads it that matters 😈 

Write a unit test with channel == "wing" and assert that source comes out as "Wing mate".

**Comment by Tkael** [comment]
A learning opportunity... I haven't written unit tests yet. :-)

**Comment by Tkael** [comment]
My master, there already exists a [unit test for messages received from wingmates](https://github.com/EDCD/EDDI/blob/99db342464f839eead746d02463ff00daa0fcdf6/Tests/JournalMonitorTests.cs#L196).
I've added a unit test for multicrew messages.

**Comment by Tkael** [comment]
After running the unit tests I think I've found and squashed the bug you saw. Please take another look.

**Pull request merged**